src/Grpc/orchestrator_service.proto

```diff
@@ -471,6 +471,15 @@ message PurgeInstancesResponse {
     google.protobuf.BoolValue isComplete = 2;
 }
 
+message RestartInstanceRequest {
```

Comment:
This should be updated once we merge PR https://github.com/microsoft/durabletask-protobuf/pull/46. But we will merge these related PR together, so I just manually updated it